profile/georgia-reborn/scripts/Base/gr-lyrics.js

```diff
@@ -121,12 +121,16 @@ class Lyrics {
 			tfilename.push(stripReservedChars($(filename)));
 		}
 
+        // extract file name from NowPlaying
+        let filePath = fb.GetNowPlaying().Path;
+        let filenameWithExtension = filePath.split('\\').pop(); // Get filename with extension
+        let filenameWithoutExtension = filenameWithExtension.replace(/\.[^.]+$/, ''); // match all up to the last dot
+
+        tfilename.push(filenameWithoutExtension);
+
 		for (let i = 0; i < tpath.length && !foundLyrics; i++) {
 			for (const file of tfilename) {
 				foundLyrics = this.checkLyrics(tpath[i], file);
-				if (foundLyrics) {
-					break;
```

Comment:
this fixes issue when `break` exists inner loop only but still going through outer loop so if your custom path lrc found you may still get false.